rasa/core/policies/ted_policy.py

```diff
@@ -280,6 +286,16 @@ def _standard_featurizer(max_history: Optional[int] = None) -> TrackerFeaturizer
             SingleStateFeaturizer(), max_history=max_history
         )
 
+    def init_split_entities(self, split_entities_config):
```

Comment:
is the same code used in DIET? could you please move it to `train_utils` to avoid code duplication